app/models/audit_activity/corrective_action/add.rb

```diff
@@ -3,6 +3,34 @@ def self.build_metadata(corrective_action)
     { corrective_action: corrective_action.attributes, document: corrective_action.document_blob&.attributes }
   end
 
+  def self.metadata_from_legacy_audit_activity(audit_activity)
+    attributes = audit_activity.body.split("<br>").each_with_object({}) do |fragment, corrective_action_attributes|
+      next if fragment.empty?
+
+      case fragment
+      when /Legislation: \*\*(.*)\*\*/
```

Comment:
my first ascii art!

Reply:
```suggestion
      when /\ALegislation: \*\*(.*)\*\*\z/
```
Ensures the match is for the full fragment rather than within a larger string.